tensorcircuit/mpscircuit.py

```diff
@@ -314,7 +576,7 @@ def mid_measurement(self, index: int, keep: int = 0) -> None:
         # normalization not guaranteed
         assert keep in [0, 1]
         self.position(index)
-        self._mps.tensors[index] = self._mps.tensors[index][:, keep, :]
+        self._mps.tensors[index] = self._mps.tensors[index][:, keep, :][:, None, :]
```

Comment:
this tensor assignment doesn't support ML backend?

Reply:
Sorry I just realized that it is not correct, will be fixed later.

Reply:
a TODO on this, this `mid_measurement` should break with ML backend even without jit, right

Reply:
done